searchgrid.py

```diff
@@ -2,6 +2,7 @@
 from collections import defaultdict as _defaultdict
 import itertools as _itertools
 
+from sklearn.compose import ColumnTransformer as _ColumnTransformer
```

Comment:
Let's make this code scikit-learn 0.19-compatible by importing this in `make_column_transformer` and skipping the corresponding tests in old versions.